### PR TITLE
Issue 1 create an entry

### PR DESCRIPTION
**Pull request by vkahlon**
https://user-images.githubusercontent.com/47346471/146878881-1d6d23be-3cc0-4254-95c0-721ab7fd7a0c.mp4

https://user-images.githubusercontent.com/47346471/146878894-62a94592-b572-4e7d-a5db-3c0ecc71bbb2.mp4

local storage
private browser result
d98af833bdb6d9f28743259ea1467dfb
Updated my CSS to reflect my style for the assignment, Font is supposed to resemble modern with a medieval touch to it.

### CODE REVIEW

**Comment by vkahlon** [comment]
UPDATED CSS ONLY, JS is still the same
https://user-images.githubusercontent.com/47346471/146982895-6fb1cf3c-b7e3-4732-a3e2-f15d7c0e1d3e.mp4

Screen Shot 2021-12-21 at 10 50 20 AM

**Comment by vkahlon** [comment]
https://user-images.githubusercontent.com/47346471/146983060-209d2fa4-3542-4334-b312-9b5806036f6c.mp4

File: js/data.js [hunk 1]
```diff
@@ -6,3 +6,12 @@ var data = {
   editing: null,
   nextEntryId: 1
 };
+window.addEventListener('beforeunload', retrieveData);
+var previousEntriesJSON = localStorage.getItem('total-entries');
+if (previousEntriesJSON != null) {
+  data.entries = JSON.parse(previousEntriesJSON);
```
Comment (codingcodymiller) [inline js/data.js:12]: You should save the entire `data` object into localStorage, not just the `entries` property

File: js/main.js [hunk 2]
```diff
@@ -1,2 +1,29 @@
 /* global data */
 /* exported data */
+function imageGenerator(event) {
+  event.preventDefault();
```
Comment (codingcodymiller) [inline js/main.js:4]: This won't be necessary here, you don't need to prevent any default behavior for the `blur` event

File: js/main.js [hunk 3]
```diff
@@ -1,2 +1,29 @@
...
+  var titleValue = $getInfoFromSubmission.elements.title.value;
+  var urlValue = $getInfoFromSubmission.elements.url.value;
+  var messageValue = $getInfoFromSubmission.elements.message.value;
+  entryInfo.title = titleValue;
+  entryInfo.url = urlValue;
+  entryInfo.message = messageValue;
+  entryInfo.entryID = data.nextEntryId;
+  data.nextEntryId++;
+  data.entries.unshift(entryInfo);
+  event.preventDefault();
```
Comment (codingcodymiller) [inline js/main.js:20]: You only need this once in this function 👀

**Review by codingcodymiller — CHANGES_REQUESTED** [review]

**Review by codingcodymiller — APPROVED** [review]

**Pull request merged**